[Docs site] Remove typo, add troubleshoot bulletin point (#8767)
cloudflare/cloudflare-docs: content/d1/learning/importing-data.md

* Ensure you are importing data in SQL format (typically with a `.sql`) file extension. See [how to convert SQLite files](#converting-sqlite-database-files) if you have a `.sqlite3` database dump.
* Make sure the schema is [SQLite3](https://www.sqlite.org/docs.html) compatible. You cannot import data from a MySQL or PostgreSQL database into D1, as the types and SQL syntax are not directly compatible.
* If you have foreign key relationships between tables, ensure you are importing the tables in the right order. You can't refer to a table that doesn't yet exist.

> * If you get "cannot start a transaction within a transaction", ensure you've removed `BEGIN TRANSACTION` and `COMMIT` from your dumped SQL statements.

Reviewer: ```suggestion
* If you get `"cannot start a transaction within a transaction"`, make sure you have removed `BEGIN TRANSACTION` and `COMMIT` from your dumped SQL statements.
```
